setup_py.patch

```diff
@@ -6,7 +6,7 @@
  
 -    setup(name='thrift',
 +    setup(name='thrift-unofficial',
-           version='0.14.0',
+           version='0.14.1',
```

Comment:
I'd slightly prefer us to tag this as `0.14.1-rc0` instead of `0.14.1`

Reply:
sure thing, done!